node/cn/api.go

```diff
@@ -177,7 +177,7 @@ func (api *PrivateAdminAPI) StopStateMigration() error {
 
 // StatusStateMigration returns the status information of state trie migration.
 func (api *PrivateAdminAPI) StatusStateMigration() map[string]interface{} {
```

Comment:
Not important but how about changing its name to `StateMigrationStatus` instead of `StatusStateMigration`?

Reply:
There is same name API. 
I will change the API name also in new PR. :)